### ISSUE TITLE
Review 'experimental process' branch (EFO:0002694)

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#959]
UPDATE: at the last single-cell curators call (May 11th 2022), @dosumis suggested to have a meeting/discussion to review and improve the branches most used by Annotare and Expression Atlas, and involving all the relevant people. We're aware that changes in the assay branch and in other terms may seriously affect their pipelines (see @anjaf's comment [here](https://github.com/EBISPOT/efo/issues/1446#issuecomment-1069234528)). Therefore, I won't do any edits for now, but I'll leave this ticket open in case it's useful for the discussion.

Some quick thoughts:

(but check [here](https://github.com/EBISPOT/efo/issues/1446#issuecomment-1069234528) before editing!)

- [ ] Review all children of EFO:0002694 'experimental process' and evaluate if any of them should be moved under the more specific OBI:0000070 'assay' (e.g. 'cap analysis gene expression'). Ideally and more broadly, this should be addressed by increasing/improving axiomatisation, so that terms will be naturally inferred to sit in a less flat hierarchy. Review is a work in progress:
- [ ] 'cap analysis gene expression' is asserted as 'part of' some 'nucleic acid library construction protocol'; this sounds incorrect, an experimental process isn't part of a protocol (which is an information entity), what should we use instead? is_about? Or rather make it part of some library preparation/construction term? Also, 'cap analysis gene expression' should be a subclass of 'assay'. Should we replace with OBI:0001674 'cap analysis of gene expression assay'? In general, now that we import OBI dynamically, do we want to start obsoleting EFO classes in favor of OBI if OBI has suitable/exact replacements, to ultimately enhance automated classification?
- [ ] Similarly, 'Deoxyribonuclease digestion' is asserted as 'part of' some 'nucleic acid library construction protocol', we should fix. And possibly replace it with OBI:0600055 'DNA restriction enzyme digestion'? Is EFO 'restriction digest' a duplicate?
- [ ] Here's a list of all other terms in the 'experimental process' branche that are asserted as 'part of' some 'nucleic acid library construction protocol' (I haven't looked at possible OBI replacements yet):
'DNA selection through 5-methylcytidine antibody'
'DNA shearing'
'hybrid selection of targets'
'hypomethylated partial restriction digest'
'methyl filtration'
'Methylation Spanning Linker Library (MSLL) processing'
'reduced representation preparation'
'restriction digest'
'size fractionation'
- [ ] 'cot filtration' is currently under 'experimental process', and that's fine though it could probably go under 'assay' more specifically (no existing OBI term). However, there are 4 more granular 'cot filtration X' terms that sit under protocol, which feels incorrect.
- [ ] 'data transformation' should go under the broader 'planned process' (same as it is in OBI) instead of 'experimental process'
- [ ] 'microRNA profiling by array' has_participant some 'DNA array' => correct to RNA array (doesn't exist yet)
- [ ] Review to be continued in alphabetical order of terms 

- [ ] The following labels are broad and could do with disambiguation, but before changing them we need to address [a potential issue with the SCEA pipeline](https://github.com/obophenotype/cell-ontology/issues/792#issuecomment-763710300):
EFO:0000487 exposure 
OBI:0600015 'group assignment' (ideally bring it up with OBI)
EFO:0000554 irradiate
EFO:0000562 labelling
EFO:0000735 washing

**Comment by paolaroncaglia** [comment]
@bvarner-ebi @zoependlington 
Following up on the single-cell curators call earlier this week. We resolved to be cautious around the 'assay' branch and any term changes that may potentially affect some EBI tools, and wait to discuss at a future meeting. So I won't do any of these edits now to play it safe. I'll remove my assignment from this ticket and hand it over to you. I also linked the ticket to the agenda for the next single-cell curators call. Thank you both.

**Comment by jamesamcl** [comment]
### Triage: still valid, still unaddressed — suggest re-scoping into an epic

I re-checked every item on this list against current `efo-edit.owl`. **All of them are still present**; the branch has not moved in the respects described since this was opened in 2021.

**Branch shape today (asserted `is_a` closure):** EFO:0002694 has **48 direct children** and **912 descendants** (892 EFO, 19 OBI, 1 BTO), none obsolete. **636** of those sit under `OBI:0000070` 'assay'. Axiomatisation is still the core problem: only **108/912** carry any relationship axiom, exactly **1** has a logical definition, and **803** have neither. 21 non-obsolete descendants have no text definition.

**Item-by-item:**

- **Children that should move under 'assay' / better axiomatisation** — not done; the 48 direct children are still flat.
- **CAGE (EFO:0004181)** — still `part_of` some EFO:0004184, still `is_a` 'experimental process' rather than 'assay'. `OBI:0001674` 'cap analysis of gene expression assay' exists and is non-obsolete, but is **not** in `obi_import.owl`.
- **Deoxyribonuclease digestion (EFO:0004186)** — unchanged. `OBI:0600055` 'DNA restriction enzyme digestion' exists, non-obsolete, not imported. EFO:0004179 'restriction digest' is still a separate sibling with the same `part_of` — the possible duplication is still open.
- **The list of 9 `part_of` protocol terms** — all 9 unchanged, and the scope is **bigger than the original list**: **20 terms** now assert `BFO:0000050` some EFO:0004184, additionally including OBI:0000415 'PCR', OBI:0000552, EFO:0004167 'RAPD', EFO:0004176 'ChIP', EFO:0004182 'RACE', EFO:0004185, EFO:0004187 and EFO:0005033 'RIP'. Whatever pattern replaces `part_of some <protocol>` should be applied to all 20 at once.
- **cot filtration** — EFO:0004455 is still a direct child of 'experimental process', and EFO:0004171–0004174 are still `is_a` EFO:0004170 'nucleic acid sequencing protocol', so the four granular terms remain detached from the parent concept.
- **'data transformation'** — `OBI:0200000` is still asserted `is_a` EFO:0002694, carrying a 21-term subtree with it.
- **'microRNA profiling by array' (EFO:0000753)** — still `has participant` some EFO:0002701 'DNA array'. No 'RNA array' or 'microRNA array' term exists yet, so this needs a new term first.
- **The 5 broad labels** (EFO:0000487, OBI:0600015, EFO:0000554, EFO:0000562, EFO:0000735) — all unchanged.

**On the blocker.** The caution in the 2022-05-13 comment traces to #1446, where @anjaf says only the **assay-branch labels** are hard-coded (Annotare); other terms are referenced by ontology ID, label changes elsewhere are fine, and deprecations are workable with notice. If that still holds, a large low-risk subset needs **no label changes at all**: fixing the 20 `part_of some <protocol>` axioms, re-parenting 'data transformation', reconnecting the cot filtration terms, and filling in the 21 missing definitions. Could a maintainer confirm whether that Annotare / Expression Atlas dependency is still live in its current form?

**Relationship to #1850.** Complementary rather than duplicate — #1850 is packaging/module extraction, this is content and axiomatisation. They interact: 636 of the 912 descendants here are assay descendants, so a split has to decide where the ~276 non-assay 'experimental process' terms go, and the re-parenting proposed here changes what any module extraction would pull. Worth sequencing deliberately.

**For the record:** this is the residue of epic #501, closed 2022-02-14 with "EFO now imports OBI dynamically, so I think we can close" — i.e. closed because the *mechanism* became available, not because the alignment was done. #12 and #54 were earlier attempts at the same review.

**Suggestion:** keep open as a tracking epic, split off the pipeline-safe items above as individual small tickets, and cross-link #1850. It would also help to settle the policy question this ticket raises but never resolves: **do we obsolete EFO classes in favour of exact OBI replacements** (CAGE → OBI:0001674, Deoxyribonuclease digestion → OBI:0600055, …) now that OBI is imported dynamically? That single decision unblocks several items.

---
*This is an automated triage review by an AI agent; maintainers, please correct if anything here is wrong.*
